CHANGELOG.md

```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
+### Framework
+
+- [Optimization] Update default rust toolchain. [@yangby]
+- [Optimization] Update cita-sdk-js version. [@kaikai]
+- [Fix] Reorganize toml path. [@WPF]
+- [Feature] Config the log module. [@kaiyu]
+
+### Executor
+
+- [Optimization] Integrate vm-test. [@WPF]
+
+### Auth
+
+- [Fix] Auth crashes when it is not ready. [@Yaorong]
+
+### Network
+
+- [Fix] Refuse duplicate nodes. [@Yaorong]
+- [Fix] Node lostes. [@Yaorong]
+- [Fix] High CPU usage. [@Yaorong]
+
+### Consensus
+
+- [Optimization] Set the default ntp service to false. [@kaikai]
+- [Fix] Not generate block. [@jerry-yu]
+
+### RPC
+
+- [Feature] Add `getVersion` interface. [@luqz]
+- [Feature] Add `peersInfoAdd ` interface. [@Yaorong]
+
+### Scripts
+
+- [Optimization] Installation && Exectution Optimization. [@clearloop]
+- [Fix] Redirect the stdout and stderr for daemon processes in docker. [@yangby]
+- [Optimization] Patch to absolute paths' in starting scripts. [@clearloop]
+- [Fix] Fix the `restart` command. [@kaikai]
+
+### Doc
```

Comment:
The doc described here has nothing to do with citahub-docs, right?

Reply:
actually, it includes the citahub-docs.
it should include all the part of CITA.

something new including the missed part, or new structure

I will add more info later

should not be added:

* the new feature description, becasue doc should be the part it